src/app/utils/index.ts

```diff
@@ -233,7 +233,7 @@ export function getConvertableDataKind(
       types = [DataKind.Categorical, DataKind.Ordinal];
       break;
     case DataType.Number:
-      types = [DataKind.Categorical, DataKind.Numerical, DataKind.Ordinal];
+      types = [DataKind.Categorical, DataKind.Numerical];
```

Comment:
Hmmm I think number can still be ordinal ?

Reply:
Agree. But we decided to exclude for numerical in the last sync meeting. @bongshin could you confirm?